### ISSUE TITLE
Fail tabs without an accessible name

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4842]
This should fail axe-core, probably under the `aria-toggle-field-name` rule:

```html
<div role="tablist">
  <span role="tab" tabindex="0">
    <i class="fa-solid fa-house"></i>
  </span>
</div>
```

**Comment by WilcoFiers** [comment]
Weird little oversight on our part.
